Patches/Dragon's Descent/Projectiles/Patch_Projectile_Animal.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- >tfw you forget to clean your teeth -->
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<!-- ==== Check for the mod. ==== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Dragon's Descent</modName>
+			</li>
+
+			<!-- ==== Projectile_DragonBreath ==== -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Projectile_DragonBreath"]/projectile</xpath>
+				<value>
+					<projectile Class="CombatExtended.ProjectilePropertiesCE">
```

Comment:
How exactly do you intend for this to work? Firing exploding projectiles, or more like a flamethrower in that it sets the surrounding area on fire?

Reply:
I imagine it acting like a flamethrower, as the amount of shots in dragons' fire breath attack is 25.